File: tidb/src/main/java/org/enodeframework/tidb/TiDBPublishedVersionStore.java
package org.enodeframework.tidb;

import org.enodeframework.jdbc.DBConfiguration;
import org.enodeframework.jdbc.JDBCPublishedVersionStore;

import javax.sql.DataSource;

/**
 * @author anruence@gmail.com
 */
public class TiDBPublishedVersionStore extends JDBCPublishedVersionStore {

    public TiDBPublishedVersionStore(DataSource dataSource) {
        super(dataSource);
    }

    public TiDBPublishedVersionStore(DataSource dataSource, DBConfiguration setting) {
        super(dataSource, setting);
    }
}

File: tidb/src/main/java/org/enodeframework/tidb/TiDBPublishedVersionStore.kt
package org.enodeframework.tidb

import org.enodeframework.jdbc.DBConfiguration
import org.enodeframework.jdbc.JDBCPublishedVersionStore
import javax.sql.DataSource

/**
 * @author anruence@gmail.com
 */
class TiDBPublishedVersionStore : JDBCPublishedVersionStore {
    constructor(dataSource: DataSource) : super(dataSource) {}
    constructor(dataSource: DataSource, setting: DBConfiguration) : super(dataSource, setting) {}
}